BuildLoader.py

```diff
@@ -266,6 +266,7 @@ def __init__(self, *args, **kwargs):
         self.HASH_STORE_SIZE       = 0x200  #Hash store size to be allocated in bootloader
 
         self.PCI_MEM64_BASE        = 0
+        self.BUILD_ARCH            = 'IA32'

```

Comment:
do you need this.  BUILD_ARCH will be initialized in line 271-272.

Reply:
BUILD_ARCH is not available until cmd_build() routine and  I want to use BUILD_ARCH in BoardConfig.py to enforce 64-bit PCI resource allocation iff it is X64 build.

Reply:
OK.  Understood now.